packages/classic-webview/src/pages/hardcore-only/UnlockHardcore.tsx

```diff
@@ -0,0 +1,28 @@
+import { PageContentContainer } from '../../components/pageContentContainer';
+import { PurchaseButton } from '../../components/PurchaseButton';
+
+export const UnlockHardcorePage = () => {
+  return (
+    <PageContentContainer showContainer={true} className="bg-[rgba(0,0,0,0.5)]">
```

Comment:
The container already has a default background.

```suggestion
    <PageContentContainer showContainer>
```